src/main/java/io/openliberty/tools/common/plugins/util/InstallFeatureUtil.java

```diff
@@ -1009,12 +1022,27 @@ private void installFeaturesOnContainer(List<String> features, boolean acceptLic
                 // The features are already installed message
                 debug(cmdResult);
             } else {
-                error("An error occurred while installing features: " + cmdResult);
+                if (isFeatureConflict(cmdResult)) {
+                    error("A feature conflict error occurred while installing features: " + features + ": "
+                            + cmdResult);
+                } else {
+                    error("An error occurred while installing features: " + cmdResult);
+                }
             }
         } else {
             // Log the successful output as debug
             debug(cmdResult);
         }
     }
-    
+
+    // Return true if feature conflict code is detected in the exception message
+    private boolean isFeatureConflict(String exceptionMessage) {
+        Pattern p = Pattern.compile(ANY_CONFLICT);
```

Comment:
You can store the Pattern in a static variable since ANY_CONFLICT does not change between invocations.

Reply:
I will address this

Reply:
Fixed in https://github.com/OpenLiberty/ci.common/pull/330